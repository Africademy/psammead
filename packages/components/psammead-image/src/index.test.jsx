import Image, { Img } from '.';
import snapshotTests from './testHelpers/snapshotTests';

describe("Image - imported as default 'Image'", () => {
  snapshotTests(Img);
});

describe("Image - Img'", () => {
  snapshotTests(Image);
});
