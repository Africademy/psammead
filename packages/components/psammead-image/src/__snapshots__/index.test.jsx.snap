// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render correctly 1`] = `
<img
  alt="Map of the UK displaying Syrian refugees and asylum seekers per 10000 population. Ranges from 0 to 17."
  className="Image-sc-7vx2mr-0 gErYCv"
  height={1067}
  src="https://ichef.bbci.co.uk/news/640/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png"
  width={853}
/>
`;
